tests/ha/crm_mon.pm

```diff
@@ -8,18 +8,20 @@
 # without any warranty.
 
 # Summary: Check cluster status in crm_mon
-# Maintainer: Denis Zyuzin <dzyuzin@suse.com>
+# Based on script by Denis Zyuzin
+# Maintainer: Loic Devulder <ldevulder@suse.com>
 
-use base "hacluster";
+use base 'hacluster';
 use strict;
 use testapi;
 use autotest;
 use lockapi;
 
 sub run {
-    type_string "crm_mon -1\n";
+    # Show cluster informations
+    script_run q(crm_mon -R -1);
```

Comment:
Yes, the two others lines use assert_script_run and I think this test should also fail if the return code is not good. I don't want to have a green test with an error!